travis/compile.sh

```diff
@@ -9,8 +9,9 @@ if [[ "$ENABLE_DEBUG" == 1 ]]; then
 else
 	DEBUG="";
 fi
-./buildconf --force
+./buildconf
```

Comment:
As @Tyrael pointed out in https://github.com/php/php-src/pull/1898#issuecomment-217811129, it seems that we do need `--force` after all. I would suggest to keep the configure-related changes out of this PR, i.e. add back --force here and drop the --config-cache option form configure.